ultravox/inference/ultravox_infer.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         model = ultravox_model.UltravoxModel.from_pretrained(
             model_path, torch_dtype=dtype
         )
+        model.to(dtype=dtype, device=device)
```

Comment:
Another bugfix: we need to move the model to device before merging LoRA weights, otherwise it's gonna be very slow.